wunderground-api.php

```diff
@@ -35,17 +32,17 @@ function main() {
         $url .= "tempf=" . convertCelsiusToFahrenheit($row["AMBIENT_TEMPERATURE"]) . "&";
         $url .= "dewptf=" . calculateDewPointF($row["AMBIENT_TEMPERATURE"], $row["HUMIDITY"]) . "&";
         $url .= "baromin=" . convertMillibarsToInches(calculateMeanSeaLevelPressure($row["AIR_PRESSURE"], Settings::$stationElevationInMeters)) . "&";
-        $url .= "soiltempf=" . convertCelsiusToFahrenheit($row["GROUND_TEMPERATURE"]) . "&";
+        if (Settings::$soilTemperatureProbePresent == 1) {
```

Comment:
Datatype mismatch: The settings are read in as strings, so the line
`if (Settings::$soilTemperatureProbePresent == 1) {`
needs to be
`if (Settings::$soilTemperatureProbePresent == "1") {`